src/quetz_client/cli.py

```diff
@@ -1,12 +1,59 @@
+import json
 import os
-from typing import Optional, cast
+from typing import Optional, Union, cast
+from urllib.parse import urlparse
 
 import fire
 from requests.adapters import HTTPAdapter, Retry
 
 from quetz_client.client import QuetzClient
 
 
+class BearerToken:
+    def __init__(self, token: str):
+        self.token = token
+
+    def __str__(self):
+        return self.token
+
+
+class ApiKey:
+    def __init__(self, token: str):
+        self.token = token
+
+    def __str__(self):
+        return self.token
+
+
+def get_auth_token(url: str, token: Optional[str]) -> Union[ApiKey, BearerToken, None]:
+    token = cast(str, token or os.getenv("QUETZ_API_KEY", ""))
+    bearer_token = os.getenv("QUETZ_BEARER_TOKEN")
+
+    if token:
+        return ApiKey(token)
+    elif bearer_token is not None:
+        return BearerToken(bearer_token)
```

Comment:
Why does `token` default to an empty string but `bearer_token` defaults to `None`? Why don't both default to `None`?